src/adapter.h

```diff
@@ -64,6 +64,8 @@ struct struct_adapter {
     uint16_t db;  // if MAX_DB then no value, else value is dB*10 of the adapter
     float strength_multiplier, // final value: strength * strength_multipler,
         snr_multiplier;        // same for snr
+    char force_strength_mode,  // 0: No force (default) 1: Force Relative 2: Force Decibel
```

Comment:
The values used here should be used as constants rather than a comment here... the constants should be used to actually decide the final value.

Reply:
Done!